collectors/rest/abuse-ch/collector.json

```diff
@@ -0,0 +1,85 @@
+{
+  "type": "collection",
+  "ttl": "4h",
+  "removeFields": [],
+  "resumeOnBoot": false,
+  "schedule": {
+    "cronSchedule": "5 * * * *",
+    "maxConcurrentRuns": 1,
+    "skippable": true,
+    "run": {
+      "rescheduleDroppedTasks": true,
+      "maxTaskReschedule": 1,
+      "logLevel": "info",
+      "jobTimeout": "0",
+      "mode": "run",
+      "timeRangeType": "relative",
+      "timestampTimezone": "UTC",
+      "expression": "true",
+      "minTaskSize": "1MB",
+      "maxTaskSize": "10MB"
+    },
+    "enabled": true
```

Comment:
Can you make this disabled please? Don't want customers to unsuspectingly start running a collector job without explicit instruction.

Reply:
Hi @bdalpe, I have disabled the schedule and updated the README accordingly.